sonar-plugin-api/src/main/java/org/sonar/api/measures/Measure.java

```diff
@@ -649,7 +644,7 @@ public final Characteristic getCharacteristic() {
   @Deprecated
   @CheckForNull
   public final Requirement getRequirement() {
-    return requirement;
+    return null;
```

Comment:
this change implies that the Measure class from the plugin api is never used to represent a base measure. It that really ok?